### PR TITLE
osc co/up: highlight pending requests' header

### PR DESCRIPTION
**Pull request by jengelh**
Found myself to mentally miss the request list of 1 pending rq and so
I am pitching the idea of subtle boldening for experimentation.

### CODE REVIEW

**Comment by marcus-h** [comment]
Sorry for the delay...

On 2021-07-10 02:43:05 -0700, Jan Engelhardt wrote:
> Found myself to mentally miss the request list of 1 pending rq and so
> I am pitching the idea of subtle boldening for experimentation.

Hmm I don't have a strong opinion here... The question is if we should
introduce a config option for disabling/enabling (I don't care about
its default) this behavior.

Any opinions?

**Comment by dmach** [comment]
I'm ok with such change if it's done differently.
I don't like 2 occurrences of  the same text and using escape codes directly.

How about to create output.py that would encapsulate all generic code working with the terminal
and move the stuff there in form of methods and constants.

output.py:
```
RESET = '\033[0m'
BOLD = '\033[1m'

def bold(text):
    if not sys.stdout.isatty():
        return text
    return BOLD + text + RESET
```

core.py:
```
msg = '\n' + output.bold('Pending requests for %s: %s (%s)')
```

File: osc/core.py [hunk 1]
```diff
@@ -7383,7 +7383,10 @@ def print_request_list(apiurl, project, package = None, states = ('new', 'review
     if not conf.config['check_for_request_on_action'] and not force:
         return
     requests = get_request_list(apiurl, project, package, req_state=states)
-    msg = 'Pending requests for %s: %s (%s)'
+    if not sys.stdout.isatty():
+        msg = '\nPending requests for %s: %s (%s)'
+    else
```
Comment (dmach) [inline osc/core.py:7388]: ... and there's missing colon after `else` -> syntax error.

Comment (dmach) [reply osc/core.py:7388]: @jengelh ping?

**Comment by dmach** [comment]
Thank you. Merging.

**Pull request merged**